.github/workflows/CheckStdlibManifest.yml

```
@@ -0,0 +1,34 @@
+name: Stdlib Manifest is up-to-date
+
+on:
+  pull_request:
+    paths:
+      - 'stdlib/*.version'
+      - 'stdlib/*/Project.toml'
+
+permissions: {}
+
+jobs:
+  check_stdlib_manifest:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Not strictly necessary in this case, but I like to make a habit of just doing it everywhere, for consistency:

```suggestion
      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
        with
        persist-credentials: false
```

---

I think you meant
```suggestion
      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
        with:
          persist-credentials: false
```
?

---

Yep, I messed up the indentation.